client/src/components/ActiveMentors/ActiveMentors.jsx

```diff
@@ -24,8 +24,8 @@ class ActiveMentors extends Component {
 				<CardText>
 					<p>
 						<FontIcon className='material-icons' style={iconStyles}>lens</FontIcon>
-						<strong>{` `}{this.state.numActive}</strong>
-						{`${this.state.numActive > 1 ?' mentors':' mentor'}
+						<strong>{` `}{this.props.user.mentors.length}</strong>
```

Comment:
You could probably store this in a variable in `render()` as to not call `.length` twice.